src/components/vm/hostdevs/hostDevAdd.scss

```diff
@@ -0,0 +1,5 @@
+.pf-c-table {
```

Comment:
You selectors needs to be more specific. Otherwise this will apply to all tables in the page.

Reply:
:100: Agreed.

I just ran across this issue with the disks table in mobile mode and spent a few minutes debugging it and found this problem.

Here's a screenshot to show that this is more than just theoretical. It's introducing bugs:

![Screen Shot 2021-08-17 at 15 36 12](https://user-images.githubusercontent.com/10246/129735354-47ac5f6d-0cd9-4393-818f-0e1918eb49d3.png)

(Notice the two scrollbars, which wouldn't be visible by default on a real mobile device. Plus, the scrolling within scrolling is a very bad pattern.)